pkg/gateway/gateway.go

```diff
@@ -23,11 +25,14 @@ func New(cfg Config) (*Gateway, error) {
 
 	sm := newServiceMapper(kc)
 
+	nginxCfg := newNGINXConfig(cfg.NGINXHealthPort)
 	var nm NGINXManager
 	if cfg.NGINXDryRun {
+		log.Printf("Creating loggingNGINXManager with NGINXConfig: %+v", nginxCfg)
```

Comment:
Agreed. I thought there was some value in logging which manager it was deciding to use but I'm not particularly opinionated on it. I've updated the existing commit.